stdlib/Random/src/RNGs.jl

```diff
@@ -582,3 +582,13 @@ function _randjump(mt::MersenneTwister, jumppoly::DSFMT.GF2X, len::Integer)
     end
     return mts
 end
+
+const TRNG = Array{MersenneTwister}
+const ThreadRNG = Ref{TRNG}()
+
+TRNG(r::MersenneTwister=GLOBAL_RNG) = TRNG(randjump(r, big(10)^20, Threads.nthreads()))
```

Comment:
It confused me a bit that you wrote `TRNG(...)` since `randjump` returns `Vector{MersenneTwister}`. It might be good to remove the typealias (e.g. `const TRNG = ...`)